Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocDateTimeValue.css

```diff
@@ -156,73 +110,23 @@ a.bocPicker12HoursWithSeconds,
 a.bocPicker24Hours,
 a.bocPicker12Hours
 {
-  position: absolute;
-  top: 0.2em;
-}
-
-span.bocDateTimeValue a.bocPicker24HoursWithSeconds,
-span.bocDateTimeValue a.bocPicker12HoursWithSeconds
-{
-  right: 6.7em;
-}
-
-.mozilla span.bocDateTimeValue a.bocPicker24HoursWithSeconds,
-.mozilla span.bocDateTimeValue a.bocPicker12HoursWithSeconds
-{
-  right: calc(6.7em + 2px);
-}
-
-span.bocDateTimeValue a.bocPicker24Hours,
-span.bocDateTimeValue a.bocPicker12Hours
-{
-  right: 5.2em;
-}
-
-.mozilla span.bocDateTimeValue a.bocPicker24Hours,
-.mozilla span.bocDateTimeValue a.bocPicker12Hours
-{
-  right: calc(5.2em + 2px);
-}
-
-span.bocDateValue a.bocPicker24Hours,
-span.bocDateValue a.bocPicker12Hours
-{
-  right: 0;
+  display: flex;
+  align-items: center;
+  height: 100%;
 }
 
 span.bocDateTimeValue a img,
 span.bocDateValue a img
 {
-  border: 0;
   text-decoration: none;
   height: 16px;
   width: 16px;
 }
 
-span.bocDateTimeValue.disabled a,
-span.bocDateValue.disabled a
-{
-  cursor: default;
-}
-
 span.bocDateTimeValue.disabled a img,
 span.bocDateValue.disabled a img
 {
   opacity: 0.4;
-  filter: progid:DXImageTransform.Microsoft.BasicImage(opacity=0.40), 
+  filter: progid:DXImageTransform.Microsoft.BasicImage(opacity=0.40),
```

Comment:
in the webbutton PR we already have a filter setup. this should be unified. (the MS-filters can go bye-bye).

There's also the question if the colors of the SVG should be done using CSS, please ask @iam-mholle.